modules/visualization/dcv-k8s-deployment/stack.py

```diff
@@ -75,41 +76,30 @@ def __init__(
             f"{dep_mod}-eks-cluster",
             cluster_name=eks_cluster_name,
             kubectl_role_arn=eks_cluster_admin_role_arn,
+            kubectl_lambda_role=iam.Role.from_role_arn(self, "KubectlHandlerArn", eks_handler_role_arn),
+            kubectl_layer=KubectlV29Layer(self, "KubectlV29Layer"),
             open_id_connect_provider=provider,
         )
 
-        namespace_manifest = {
```

Comment:
Synced offline - this shouldn't be deleted it seems

Reply:
tested e-e. No longer needed.